server/services/messaging_service_test.go

```diff
@@ -361,6 +361,36 @@ func Test_Blurb_lifecycle(t *testing.T) {
 	if err != nil {
 		t.Errorf("Create: unexpected err %+v", err)
 	}
+	// create a Blurb with legacy_user_id
+	second, err := s.CreateBlurb(
+		context.Background(),
+		&pb.CreateBlurbRequest{
+			Parent: "users/rumble/profile",
+			Blurb: &pb.Blurb{
+				User:    "users/rumble",
+				Content: &pb.Blurb_Text{Text: "non-slash resource test."},
+				LegacyUserId: "legacy_rumble",
+			},
+		})
+	if err != nil {
+		t.Errorf("Create: unexpected err %+v", err)
+	}
+	// get the second Blurb from database and verify the legacy_user_id, then delete it.
+	gotSecond, err := s.GetBlurb(
+		context.Background(),
+		&pb.GetBlurbRequest{Name: second.GetName()})
+	if err != nil {
+		t.Errorf("Get: unexpected err %+v", err)
+	}
+	if !proto.Equal(second, gotSecond) {
+		t.Error("Expected to get created blurb.")
+	}
+	_, err = s.DeleteBlurb(
+		context.Background(),
+		&pb.DeleteBlurbRequest{Name: second.Name})
```

Comment:
```suggestion
		&pb.DeleteBlurbRequest{Name: second.GetName()})
```

Reply:
Thanks! I'm wondering what's the difference between `second.Name` and `second.GetName()` ? 
In the below delete test L408, it calls the Blurb's name as `&pb.DeleteBlurbRequest{Name: delete.Name})`🤓

Reply:
As I mentioned earlier, we want to use the Getters to access the value of a message. This _can_ protect us from rogue `nil` pointer panics (e.g.  if `second` was `nil`, `GetName()` returns empty string rather than crashing as it would with a direct field access on a `nil`). The other thing is that since proto3_optional has been introduced, using the Getters protect us from accidentally breaking ourselves - i.e. adding `optional` to `Blurb.name` would change the underlying type to `*string` from `string`, which might not match the `DeleteBurbRequest.name` which isn't `optional` and is just a `string`. I hope that makes some sense.

The previously written code should use the Getter too, so let's not continue doing the more risky thing :) Don't worry about changing that code in this change, maybe we can refactor some stuff at a later date.

Reply:
Sounds very reasonable to me, changing it to Getters now, thanks!